$NetBSD: patch-ad,v 1.8 2015/01/28 03:07:24 mef Exp $

Perl in pkgsrc doesn't install or tweak perllocal.pod, so comment
out that test.

--- t/INSTALL_BASE.t.orig	2015-01-10 01:41:54.000000000 +0900
+++ t/INSTALL_BASE.t	2015-01-28 11:52:19.000000000 +0900
@@ -21,7 +21,7 @@ use Test::More
     !MM->can_run(make()) && $ENV{PERL_CORE} && $Config{'usecrosscompile'}
     ? (skip_all => "cross-compiling and make not available")
     : ();
-plan tests => 4 + $CLEANUP + @INSTDIRS * (15 + $CLEANUP);
+plan tests => 4 + $CLEANUP + @INSTDIRS * (15 + $CLEANUP) - 4;
 
 my $Is_VMS = $^O eq 'VMS';
 
@@ -71,7 +71,7 @@ for my $instdir (@INSTDIRS) {
     ("$instdir/lib/perl5/Big/Dummy.pm",
      "$instdir/lib/perl5/Big/Liar.pm",
      "$instdir/bin/program",
-     "$instdir/lib/perl5/$Config{archname}/perllocal.pod",
+     #"$instdir/lib/perl5/$Config{archname}/perllocal.pod",
      "$instdir/lib/perl5/$Config{archname}/auto/Big/Dummy/.packlist"
     );
 
